### PR TITLE
Update to PayPal Checkout v0.55.0

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

- Update PayPalCheckout SDK to [v0.55.0](https://github.com/paypal/paypalcheckout-ios/releases/tag/0.55.0)
- Fix structure of tokenization request & pass in correlation ID from PayPalCheckout SDK
- Update demo view controller to enable button when flow completes & retain an instance of `BTPayPalNativeClient`

### Checklist

- ~Added a changelog entry~

- @sestevens 
- @sshropshire

### CODE REVIEW

File: Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeTokenizationClient.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ class BTPayPalNativeTokenizationClient {
 
         let tokenizationRequest = BTPayPalNativeTokenizationRequest(returnURL: returnURL,
                                                                     request: request,
-                                                                    correlationID: "", // TODO: get this value from PayPalCheckout SDK
+                                                                    correlationID: State.correlationIDs.riskCorrelationID ?? "",
```
Comment (sestevens) [inline Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeTokenizationClient.swift:19]: Question: Is defaulting to an empty string appropriate here or should we return an error to the merchant if we don't have a correlation ID at this point?

**Comment by sestevens** [comment]
Oops, looks like we forgot to update the unit tests. Moving this back to draft.

File: Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeClient.swift [hunk 2]
```diff
@@ -48,9 +48,14 @@ import PayPalCheckout
                                                                        environment: order.environment)
 
                 PayPalCheckout.Checkout.set(config: payPalNativeConfig)
+                PayPalCheckout.Checkout.showsExitAlert = false
```
Comment (scannillo) [inline Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeClient.swift:51]: 👏

File: Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeTokenizationClient.swift (hunk 1 above)
Comment (scannillo) [reply Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeTokenizationClient.swift:19]: Didn't we talk about this in stand? We decided to go with erroring if there is no `riskCorrelationID` from the Native XO right? @sshropshire can you confirm?

Comment (scannillo) [reply Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeTokenizationClient.swift:19]: I added a TODO for this and am going to add a JIRA for this work. `State.correlationIDs.riskCorrelationID` is proving to be pretty tricky to mock.

**Review by scannillo — APPROVED** [review]
Added JIRA for additional testing todo

**Pull request merged**